gcloud-java-dns/src/main/java/com/google/gcloud/dns/ChangeRequest.java

```diff
@@ -148,10 +148,34 @@ public Dns dns() {
   }
 
   /**
-   * Applies this change request to the associated zone.
+   * Applies this change request to the zone identified by {@code zoneName}.
    */
-  public ChangeRequest applyTo(Dns.ChangeRequestOption... options) {
-    return dns.applyChangeRequest(zone, this, options);
+  public ChangeRequest applyTo(String zoneName, Dns.ChangeRequestOption... options) {
+    return dns.applyChangeRequest(zoneName, this, options);
+  }
+
+  /**
+   * Retrieves the up-to-date information about the change request from Google Cloud DNS. Parameter
```

Comment:
I think it's worth mentioning that objects of this class are immutable (this could go to the class' javadoc) and that this method returns an object containing updated information.